{"name": "test_negative_order_number_get_order_request", "status": "passed", "steps": [{"name": "Check get order request with wrong order number return error", "status": "passed", "start": 1701261853025, "stop": 1701261853025}], "start": 1701261853025, "stop": 1701261853025, "uuid": "791b5f24-27dd-43c3-80ee-d61bda81165f", "historyId": "069910fcc44cbb6eef8609d9b375d91b", "testCaseId": "069910fcc44cbb6eef8609d9b375d91b", "fullName": "tests.test_get_order.TestCreateCourier#test_negative_order_number_get_order_request", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "MacBook-Air-Aleksej.local"}, {"name": "thread", "value": "57046-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order"}]}